### PR TITLE
Disable batch norm by default, and reword CLI argument

### PR DESCRIPTION
**Pull request by jonwzheng**
## Description
We recently decided to turn off batch norm by default. 
This small PR:
1. Sets batch norm off by default for `MPNN` and `MultiComponentMPNN`
2. Changes the CLI argument `--no-batch-norm` to `--batch-norm` and accordingly adjusts how the argument is handled when building the MPNN.

### CODE REVIEW

**Comment by jonwzheng** [comment]
The changes to the `.ipynb` notebooks are because they were initially run with the default batch_norm (batch_norm=True). Reran them with the default `False` setting now unless they are loaded from a checkpoint or `batch_norm` was set to `True` explicitly.

File: docs/source/tutorial/python/models/basic_mpnn_model.ipynb [hunk 1]
```diff
@@ -142,7 +142,7 @@
     }
    ],
    "source": [
-    "MPNN(mp, agg, ffn, batch_norm=False)"
+    "MPNN(mp, agg, ffn, batch_norm=True)"
```
Comment (shihchengli) [inline docs/source/tutorial/python/models/basic_mpnn_model.ipynb:145]: I think the original file is okay as `batch_norm` is explicitly set to `False`. I am fine with this change though.

**Review by shihchengli — COMMENTED** [review]
The output for the cell of running `trainer.fit` seems a bit weird. Let me know if you need my help to rerun them.

File: docs/source/tutorial/python/models/basic_mpnn_model.ipynb (hunk 1 above)
Comment (jonwzheng) [reply docs/source/tutorial/python/models/basic_mpnn_model.ipynb:145]: The purpose of this change was because in the original notebook, we say "It is on by default but can be turned off" and explicitly turn the `batch_norm` flag to `False`. Since we are inverting it, now the text says "It is off by default but can be turned on.", I think we should explicitly set the flag to `True`; my understanding is that this cell is to demonstrate how to change the `batch_norm` flag.

**Comment by jonwzheng** [comment]
> The output for the cell of running `trainer.fit` seems a bit weird. Let me know if you need my help to rerun them.

Yes, if you could please do this. I'm running the cells on Ubuntu which might format things strangely.

**Review by shihchengli — APPROVED** [review]

**Review by KnathanM — APPROVED** [review]
I will also update this in the v1 to v2 user transition [guide](https://docs.google.com/spreadsheets/u/3/d/e/2PACX-1vRshySIknVBBsTs5P18jL4WeqisxDAnDE5VRnzxqYEhYrMe4GLS17w5KeKPw9sged6TmmPZ4eEZSTIy/pubhtml#).
